pages/admin.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>CellSwiper</title>
+    <meta charset="UTF-8"/>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+
+    <!-- Bootstrap -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
+            integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
+            crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="../styles/theme.css">
+    <link rel="stylesheet" href="../styles/admin.css">
+    <script src="../scripts/admin.js"></script>
+</head>
+<body>
+    <div class="row nav">
+        <div class="col-xs-6 col-sm-6 col-md-offset-1 col-md-2">
+            <h3 id="logo">CellSwiper</h3>
+        </div>
+        <div class="col-xs-6 col-sm-6 col-md-offset-6 col-md-2" id="account-info">
+            <h3 id="name"></h3>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-xs-12 col-sm-12 col-md-offset-1 col-md-5" id="image-dir">
```

Comment:
col-* classes implicitly apply to all above sizes, so the col-sm-12 is unnecessary because it is implied by the col-xs-12. It doesn't hurt anything to have it in, but just for future reference